clang/unittests/Analysis/FlowSensitive/TypeErasedDataflowAnalysisTest.cpp

```diff
@@ -146,6 +146,30 @@ TEST_F(DataflowAnalysisTest, DiagnoseFunctionDiagnoserCalledOnEachElement) {
                                    " (Lifetime ends)\n")));
 }
 
+TEST_F(DataflowAnalysisTest, CanAnalyzeStmt) {
+  std::string Code = R"cc(
+      struct S {int i;};
+      S getAnS() {return S{1};};
```

Comment:
Why do we need this function instead of simply initializing `AnS` directly with `S{1}` as in the test above?

Reply:
Just an ever so slightly more interesting analysis to test. Simplified now so that we can check the field value.